python/code_challenges/stack_queue_brackets.py

```diff
@@ -1,5 +1,32 @@
-from data_structures.queue import Queue
+# from data_structures.queue import Queue
+from data_structures.stack import Stack
+
+
+def multi_bracket_validation(value):
+    '''
+    Write a function called validate brackets
+    Arguments: string
+    Return: boolean
+    representing whether or not the brackets in the string are balanced
+    '''
+    stack = []
+    for data in value:
+        if data == '(' or data == '[' or data == '{':
```

Comment:
If you moved these into a front_brackets variable of type list, string, or dict you could change this to 
```
if char in front_brackets:
         stack.append(data)
```